lib/bindings/langs/android/lib/src/main/kotlin/breez_sdk_liquid_notification/job/SwapUpdated.kt

```diff
@@ -57,8 +61,9 @@ class SwapUpdatedJob(
 
     override fun onEvent(e: SdkEvent) {
         when (e) {
-            is SdkEvent.PaymentWaitingConfirmation -> handlePaymentEvent(e.details)
-            is SdkEvent.PaymentSucceeded -> handlePaymentEvent(e.details)
+            is SdkEvent.PaymentWaitingConfirmation -> handlePaymentSuccess(e.details)
+            is SdkEvent.PaymentSucceeded -> handlePaymentSuccess(e.details)
+            is SdkEvent.PaymentWaitingFeeAcceptance -> handlePaymentWaitingFeeAcceptance(e.details)
```

Comment:
The comment here is not related to this PR but I think we might miss the notification if the app keeps working in the background and we have two sdk instances as one instance will emit it and the notifications job instance will miss it?
Perhaps we should record the state when we get the notification and poll for changes in addition to waiting for events?
@dangeross what do you think?
Even if this is indeed something we need to fix I suggest not to handle that as part of this PR as it was introduced before.

Reply:
If the swap is already WaitingFeeAcceptance I guess the swap isn't tracked (last swap status received in loop) when the notification SDK instance is started? If so I think you're right, we should fetch the payment by the hashed swap id, then check it's state and/or wait for events.

Reply:
@roeierez I created an [issue](https://github.com/breez/breez-sdk-liquid/issues/636) to track this. I haven't seen this in practice, but I usually stop the app when testing the notification, so I wouldn't notice it.